src/Ens/JobeetBundle/DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,29 @@
+<?php
```

Comment:
@PhotonSapiens 
I cannot comment `src/Ens/JobeetBundle/DataFixtures/ORM/LoadJobData.php~` as far as it is empty - but this is another temp file you should not commit